src/api/membersApi.ts

```diff
@@ -0,0 +1,8 @@
+import { MemberType } from '../model/Member';
+import { getSupabaseClient } from './supabase';
+
+export const getMembers = async () => {
+  const { data } = await getSupabaseClient().from('members').select('*');
```

Comment:
We dont need the asterisk, it is implicitly there if you leave it empty I believe. 
```suggestion
  const { data } = await getSupabaseClient().from('members').select();
```